libraries/aws/shadow/cbmc/proofs/Shadow_GetTopicString/Shadow_GetTopicString_harness.c

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (C) 2020 Amazon.com, Inc. or its affiliates.  All Rights Reserved.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy of
+ * this software and associated documentation files (the "Software"), to deal in
+ * the Software without restriction, including without limitation the rights to
+ * use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of
+ * the Software, and to permit persons to whom the Software is furnished to do so,
+ * subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+ * FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
+ * COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
+ * IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+ * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+/**
+ * @file Shadow_GetTopicString_harness.c
+ * @brief Implements the proof harness for Shadow_GetTopicString function.
+ */
+
+#include "shadow.h"
+#include "shadow_cbmc_state.h"
+
+void harness()
+{
+    char * topicBuffer;
+    uint16_t bufferSize;
+    uint16_t * pOutLength;
+    char * pThingName;
+    uint8_t thingNameLength;
+    uint8_t topicType;
+
+    topicBuffer = mallocCanFail( bufferSize );
+    pOutLength = mallocCanFail( sizeof( uint16_t ) );
+    pThingName = mallocCanFail( thingNameLength );
+
+    Shadow_GetTopicString( topicType,
```

Comment:
Does this return anything? Anything we can assert should be true after it finishes?

Reply:
This is the typical workflow for our proofs. We do not check the return values.

Reply:
I will put this into SIM so that we will do that moving forward with new proofs and also to go back and update the existing.  Thank yo for the insight.

Reply:
Sounds good to me